README.md

```diff
@@ -8,3 +8,48 @@ This project is not setup for contributing code. It is intended only for trackin
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# Azure Container Compute Upstream Projects
+
+This list of projects is maintained by the Azure container compute upstream team (TODO: how to contact the team). This list is inteded to help you make informed decisions about what projects to use (or not use) in the context of your goals (e.g. proof of concept vs. production). To make this decision you need to consider your goals, your need for formal support, the project's [maturity](#Maturity), governance, version level, and your willingness to work in open source.
```

Comment:
Any ideas on the best public way for us to be contacted? Use the issues queue on this project? Create a DL that can get external emails?

Reply:
Right now I'm leaning towards GitHub issues, and actually maintaining them properly

Reply:
You'll probably see a lot of issues in this repo, which are related to the actual projects.

Reply:
I've tweaked the language in support to mitigate that, and also I can move issues between repos if needed - will help me get visibility to those issues too :)